debian/syncd.install

```diff
@@ -1,3 +1,4 @@
 usr/bin/saidump
 usr/bin/saiplayer
 usr/bin/syncd*
+syncd/usr/bin/* usr/bin
```

Comment:
It's up to you. But I think making hierarchy syncd/usr/bin/ is not required. Probably it's better make it syncd/scripts? Make or not make bin_SCRIPTS is up to you.